tsconfig.json

```diff
@@ -0,0 +1,16 @@
+{
+  "compilerOptions": {
+    "declaration": true,
+    "experimentalDecorators": true,
```

Comment:
Are we using decorators anywhere? Is there any compelling reason to turn them on by default?

I've been burned once before when they basically re-wrote the decorators spec. I'm not sure we should encourage their use before they reach stage 3 🤔

Reply:
We are not, and I (shamefully) copied this from GABA